cumulusci/core/flowrunner.py

```diff
@@ -359,6 +359,7 @@ def run(self, org_config):
                     jinja2_context = {
                         "project_config": step.project_config,
                         "org_config": self.org_config,
+                        "StrictVersion": StrictVersion,
```

Comment:
fwiw it looks like StrictVersion implements comparison in a way that will cast compared strings to StrictVersion. So we could probably do this without having StrictVersion in the expression context:

    "npsp" in org_config.installed_packages and org_config.installed_packages["npsp"] <= "3.175"

but that's still a bit of a mouthful; maybe OrgConfig should have a has_sufficient_package_version(namespace, minimum_version) method?